dvc/config.py

```diff
@@ -109,6 +109,7 @@ class Config(object):  # pylint: disable=too-many-instance-attributes
     SECTION_CORE_ANALYTICS = "analytics"
     SECTION_CORE_CHECKSUM_JOBS = "checksum_jobs"
     SECTION_CORE_HARDLINK_LOCK = "hardlink_lock"
+    SECTION_CORE_SUBREPO = "subrepo"
```

Comment:
Do we really need this "signal" in the config? @pared I'd appreciate if you could explain?

Reply:
I needed it at the beginning, but thanks to @efiop review it got obsolete, removing